css/properties/margin-block-end.json

```diff
@@ -6,15 +6,51 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/margin-block-end",
           "spec_url": "https://drafts.csswg.org/css-scroll-snap/#scroll-padding",
           "support": {
-            "chrome": {
-              "version_added": "69"
-            },
-            "chrome_android": {
-              "version_added": "69"
-            },
-            "edge": {
-              "version_added": "79"
-            },
+            "chrome": [
+              {
+                "version_added": "87"
+              },
+              {
+                "version_added": "69",
```

Comment:
You're overwriting the exiting version 69 data. Confirmed that this was correct from https://bugs.chromium.org/p/chromium/issues/detail?id=850004#c2.

Reply:
<https://github.com/chromium/chromium/commit/c2ffe93299c49e5d6c6db6d1df34fa1f6c713913> says that it’s behind a flag though.

Reply:
Confirming version number with https://chromestatus.com/feature/6618771051511808 as well.  👍